lib/harvest/user.rb

```diff
@@ -47,10 +47,12 @@ def timezone=(timezone)
       when 'mst', 'mdt' then self.timezone = 'america/denver'
       when 'pst', 'pdt' then self.timezone = 'america/los_angeles'
       else
-        if Harvest::Timezones::MAPPING[tz]
-          self["timezone"] = Harvest::Timezones::MAPPING[tz]
+        self["timezone"] = if Harvest::Timezones::MAPPING[tz]
```

Comment:
`self.timezone=` seems to be a recursive call. Also, this code now maps both ways and I don't think thats the intended behavior.